ui/src/pages/version/components/forms/components/transformer/components/simulation/TransformerSimulationInput.js

```diff
@@ -11,19 +12,50 @@ import {
 import React from "react";
 import { Panel } from "../../../Panel";
 import { simulationIcon } from "./constants";
+import {PROTOCOL} from "../../../../../../../../services/version_endpoint/VersionEndpoint"
+import {appConfig} from "../../../../../../../../config"
 
 export const TransformerSimulationInput = ({
   simulationPayload,
+  protocol,
   onChange,
   onSubmit,
   errors
 }) => {
+  const populateSampleRequest = () => {
+    const defaultPayload = {"target_name":"probability","prediction_table":{"name":"prediction_table_sample","columns":[{"name":"col1","type":"TYPE_INTEGER"},{"name":"col2","type":"TYPE_DOUBLE"}],"rows":[{"rowId":"row1","values":[{"integer_value":1},{"doubleValue":4.4}]},{"rowId":"row2","values":[{"integer_value":2},{"doubleValue":8.8}]}]},"transformer_input":{"tables":[{"name":"sample_table","columns":[{"name":"col1","type":"TYPE_INTEGER"},{"name":"col2","type":"TYPE_DOUBLE"}],"rows":[{"rowId":"row1","values":[{"integer_value":1},{"double_value":4.4}]},{"rowId":"row2","values":[{"integer_value":2},{"double_value":8.8}]}]}],"variables":[{"name":"var1","type":"TYPE_INTEGER","integer_value":1},{"name":"var2","type":"TYPE_DOUBLE","double_value":1.1}]},"prediction_context":[{"name":"ctx1","type":"TYPE_INTEGER","integer_value":1},{"name":"ctx2","type":"TYPE_STRING","double_value":"SAMPLE"}]}
```

Comment:
so the  generated payload is "fixed"? Are we able to generate one that will actually work based on the yaml?

Reply:
the aims to give user clue how the valid structure of request will look like, this is something that we can improve in the next iteration